src/main/resources/openapi/tasks.yaml

```diff
@@ -132,6 +132,8 @@ components:
           $ref: '#/components/schemas/CronExpression'
         is_active:
           $ref: '#/components/schemas/IsActive'
+        batchSize:
```

Comment:
Note the `AutomatedTaskSummary` object is also used by the `GET /admin/automated-tasks` endpoint which returns all automated tasks, whereas DMP-3915 only specifies changing the `GET /admin/automated-tasks/{id}` endpoint, so this change will have broader consequences than what the ticket intends.

Suggest this be moved into the `DetatailedAutomatedTask` object instead, which is only used by the get by id endpoint?

Also the naming convention for API properties is snake_case (see https://hmcts.github.io/restful-api-standards/#118), so the attribute name on L135 should be `batch_size`.